### PR TITLE
Raagwadh/vz-11040 Add Ons

### PR DESCRIPTION
**Pull request by Raaghavcodes**
Add on stuff to VZ-11040
- CommitID.txt created on first run
- newimagefound.txt created on first run
- Artifact added
- CommitID

### CODE REVIEW

File: ci/scripts/check_image_sizes.sh [hunk 1]
```diff
@@ -39,6 +39,8 @@ oci --region us-phoenix-1 os object get --namespace ${OCI_OS_NAMESPACE} -bn ${OC
 if [ $? -ne  0 ] ; then
  echo "${CLEAN_BRANCH_NAME}/image-list not found"
  oci --region us-phoenix-1 os object put --force --namespace ${OCI_OS_NAMESPACE} -bn ${OCI_OS_BUCKET} --name ${CLEAN_BRANCH_NAME}/image-list --file ${WORKSPACE}/image-sizes.txt
+echo "${GIT_COMMIT}">${WORKSPACE}/commitID.txt
+echo " ">${WORKSPACE}/newimagefound.txt
```
Comment (bobdonat) [inline ci/scripts/check_image_sizes.sh:43]: creating the newimagefound.txt file is not needed in this case

Comment (Raaghavcodes) [reply ci/scripts/check_image_sizes.sh:43]: I ran into an error where it "could not find newimagefound.txt file"
Thats why I did that

Comment (Raaghavcodes) [reply ci/scripts/check_image_sizes.sh:43]: As can be seen here:
https://build.verrazzano.io/job/verrazzano-distributions-testing/job/raagwadh%252FVZ-11040-add-on/4/execution/node/83/log/

Comment (bobdonat) [reply ci/scripts/check_image_sizes.sh:43]: ok, not sure I understand why

**Review by bobdonat — APPROVED** [review]

**Pull request merged**